### ISSUE TITLE
微调clip为啥这么难

### ISSUE THREAD

**Issue by xuboot** [OFA-Sys/Chinese-CLIP#146]
跑都跑不起来

**Comment by yangapku** [comment]
您好，不应该呀😂我们的示例流程已经非常细致了，tianchi参赛同学用了都说好。请问有没有在MUGE电商数据集上，先用我们readme里面提供的notebook试着跑跑，上上手？具体卡在哪一步了呢

**Comment by xuboot** [comment]
谢谢大哥，那如果我自己的数据呢，怎么适配这个数据预处理呢，我就想简单的微调几条数据，然后进行测试，测试出自己加进去的数据集，我输入一只卡通青龙的时候，他返回的是一条同时我的图片

**Comment by xuboot** [comment]
@yangapku main.py: error: unrecognized arguments: --local_rank=0
ERROR:torch.distributed.elastic.multiprocessing.api:failed (exitcode: 2) local_rank: 0 (pid: 1883742) of binary: /usr/local/data/conda-env/conda/bin/python3  大哥我这个为啥啊，我跑的是你给我的例子然后目前的环境是单卡，a10跑的

**Comment by xuboot** [comment]
@yangapku rdteam   1884103 1884068  0 11:46 pts/8    00:00:00 /bin/bash -c export PYTHONPATH=${PYTHONPATH}:`pwd`/cn_clip;python3 -m torch.distributed.launch --nproc_per_node=1 --nnodes=1 --node_rank=1       --master_addr=localhost --master_port=8514 cn_clip/training/main.py       --train-data=../datapath/datasets/MUGE/lmdb/train       --val-data=../datapath/datasets/MUGE/lmdb/valid       --num-workers=1       --valid-num-workers=1       --resume=../datapath/pretrained_weights/clip_cn_vit-b-16.pt       --reset-data-offset       --reset-optimizer       --logs=../datapath/experiments/       --name=muge_finetune_vit-b-16_roberta-base_bs48_1gpu       --save-step-frequency=999999       --save-epoch-frequency=1       --log-interval=10       --report-training-batch-acc       --context-length=52       --warmup=100       --batch-size=24       --valid-batch-size=24       --valid-step-interval=1000       --valid-epoch-interval=1       --lr=3e-06       --wd=0.001       --max-epochs=1       --vision-model=ViT-B-16       --use-augment       --grad-checkpointing       --text-model=RoBERTa-wwm-ext-base-chinese 
rdteam   1884105 1884103  3 11:46 pts/8    00:00:01 python3 -m torch.distributed.launch --nproc_per_node=1 --nnodes=1 --node_rank=1 --master_addr=localhost --master_port=8514 cn_clip/training/main.py --train-data=../datapath/datasets/MUGE/lmdb/train --val-data=../datapath/datasets/MUGE/lmdb/valid --num-workers=1 --valid-num-workers=1 --resume=../datapath/pretrained_weights/clip_cn_vit-b-16.pt --reset-data-offset --reset-optimizer --logs=../datapath/experiments/ --name=muge_finetune_vit-b-16_roberta-base_bs48_1gpu --save-step-frequency=999999 --save-epoch-frequency=1 --log-interval=10 --report-training-batch-acc --context-length=52 --warmup=100 --batch-size=24 --valid-batch-size=24 --valid-step-interval=1000 --valid-epoch-interval=1 --lr=3e-06 --wd=0.001 --max-epochs=1 --vision-model=ViT-B-16 --use-augment --grad-checkpointing --text-model=RoBERTa-wwm-ext-base-chinese

**Comment by xuboot** [comment]
@yangapku 一直卡在那里不动，然后cpu ,gpu一点都不动为啥啊，大哥

**Comment by xuboot** [comment]
@yangapku 
![095432e1a5da2b2c12534fa41b60988](https://github.com/OFA-Sys/Chinese-CLIP/assets/54123895/245741af-d629-40d0-a493-daa2e8a21acd)
